app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -30,16 +30,20 @@
       <% end %>
     <% end %>
     <% if history.any? %>
-      <div class="change-notes" id="history">
-        <h3>Page history</h3>
-        <ol>
-          <% history.each do |change| %>
-            <li>
-              <time datetime="<%= change[:timestamp] %>" class="timestamp"><%= change[:display_time] %></time>
-              <%= change[:note] %>
-            </li>
-          <% end %>
-        </ol>
+      <div class="change-notes" data-module="toggle">
+        <p>
+          <a href="#full-history" data-controls="full-history" data-expanded="false">+ full page history</a>
```

Comment:
Discussed offline with @fofr: we've decided to keep the same copy and behaviour from Whitehall, for now. We can amend it easily if need be.